packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -17,6 +21,10 @@ export interface GridCellParams {
    * The cell value, but if the column has valueGetter, use getValue.
    */
   value: GridCellValue;
+  /**
+   * The cell value formatted with the column valueFormatter.
+   */
+  formattedValue: GridCellValue;
```

Comment:
How has this new prop something to do with the change of the event logic?

Reply:
Why not work on #273 in a different PR?

Reply:
Well the goal for me was also to clean up all the params that are passed in events and around, so it all comes from a single source and all homogeneous. But yeah I could have treated this one separately.